### ISSUE TITLE
Surface_Tracker_Offline.cleanup() crashes if it has active export jobs

### ISSUE THREAD

**Issue by papr** [pupil-labs/pupil#1847]
Traceback
```py
Traceback (most recent call last):
  File "/Users/papr/work/pupil/pupil_src/launchables/player.py", line 679, in player
    g_pool.plugins.clean()
  File "/Users/papr/work/pupil/pupil_src/shared_modules/plugin.py", line 396, in clean
    p.cleanup()
  File "/Users/papr/work/pupil/pupil_src/shared_modules/surface_tracker/surface_tracker_offline.py", line 560, in cleanup
    for proxy in self.export_proxies:
RuntimeError: Set changed size during iteration
```

**Comment by pfaion** [comment]
Note: This happens when the surface tracker is closed while the export is running, e.g. during the closing of Pupil Player or by disabling the plugin.

**Comment by pfaion** [comment]
Fixed with #1848
